src/xunit.execution/Sdk/ExtensibilityPointFactory.cs

```diff
@@ -39,7 +39,12 @@ static object CreateInstance(IMessageSink diagnosticMessageSink, Type type, obje
 
             var disposable = result as IDisposable;
             if (disposable != null)
-                disposalTracker.Add(disposable);
+            {
```

Comment:
Coding style for this project is to not use curly braces for single statement bodies (that would include both the `if` and the `lock` here).